app/models/setting/provisioning.rb

```diff
@@ -15,6 +15,23 @@ def self.local_boot_labels
   Setting::BLANK_ATTRS.push(*(default_global_labels + local_boot_labels))
   validates :value, :pxe_template_name => true, :if => Proc.new { |s| s.class.default_global_labels.include?(s.name) }
 
+  IGNORED_INTERFACES = [
```

Comment:
I'm not sure if this is an act of rebellion against hound-ci, bu having this on one or two lines (based on line length) would be sufficient for me :)

Reply:
@lzap do you insist on this form of the array definition (compared to splitting it just to 2 lines?) I will not block the PR,
but want to see at least the response before we proceed

Reply:
Yeah sorry late on GitHub, I like one entry per line, we have been editing this recently a lot and this is much more readable, sortable and visible.

Not sure what the next action should be, it passed tests and I think squash is the way to go?